features/operators/negative/does-not-include.feature

```diff
@@ -0,0 +1,58 @@
+Feature: compare JSON does not include X
```

Comment:
you should add compound operand tests for `does not include`, stuff like:

```gherkin
{ "a": 1 } does not include { "a": 1, "b": 2 }
```
:point_up: also what would be the right outcome of this? `{ "a": 1 }` certainly doesn't include the full `{ "a": 1, "b": 2 }` but it does include a part of it.

maybe we also need a `does not overlap` operator that distinguishes between inclusion and intersection.